### ISSUE TITLE
MSWord:  NVDA  stops responding on a document that has table with number of rows more than 400.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4954]
**Reported by sumandogra on 2015-02-27 05:24**
When one opens a document with a large table or tries to make a table that has a large number of rows more than 400, NVDA stops responding in that particular document.  On ALT+TABBING away, NVDA functions normally, but on returning back, NVDA stops responding.  
There is no speech output.  This problem was not present in NVDA 2014.4.
Steps to reproduce:
1.  Open a document that has a table with 550 rows.  Alternatively, one can make a document.
2.  To create a document of this sort, go to Insert tab and then go to tables.
3.  Press SPACEBAR on the tables grouping to drop a menu. Go to Insert a table menu command and press ENTER.
4.  TAB to number of rows and input a value more than 500.
5.  Press ENTER to close the dialog.
6.  Now try to read the document.  NVDA does not respond.
7.  ALT+TAB away and come back.  There is no r esponse.

**Comment by bhavyashah** [comment]
Partially confirmed using Word 2010 and NVDA 2017.3 on a Windows 8.1 system. Let me summarize the STR employed, results experienced and other information I would like to share.
### STR ###
* Enable cell all table reporting options (tables, cell coordinates and row/column headers) in NVDA's Document Formatting settings dialog.
* Open a blank Word document.
* Create a table (Alt, N, T, I) of 500 rows and 63 columns.
* Arrow around or Ctrl+Alt+arrow around in the table.
Expected Result: NVDA does what it usually does in tables with minimal delay.
Actual Result: NVDA does what it usually does, i.e. reporting the cell coordinate of the cell to which the user has moved, just with a constant delay of 1-1.5 seconds.

In other words, I am unable to reproduce any NVDA or Word freeze or crash, and NVDA does work as expected with all table reporting options enabled, but just with excessive latency.
### Log Snippet ###
IO - inputCore.InputManager.executeGesture (19:58:14.805):
Input: kb(laptop):downArrow
DEBUG - editableText.EditableText._hasCaretMoved (19:58:14.997):
Caret move detected using bookmarks. Elapsed: 0 ms
IO - speech.speak (19:58:16.341):
Speaking [LangChangeCommand ('en_GB'), u'row 2']

**Comment by derekriemer** [comment]
A 1.5 second "freeze" of NVDA is what you are seeing, right?

**Comment by bhavyashah** [comment]
I would probably call it a delay because NVDA does report the newly
focused cell properly after 1.5 seconds, but I suppose one could view
it as a freeze also.

On 10/13/17, Derek Riemer <notifications@github.com> wrote:
> A 1.5 second "freeze" of NVDA is what you are seeing, right?
>
> --
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/4954#issuecomment-336411415


-- 
Best Regards
Bhavya Shah

Blogger at Hiking Across Horizons: https://bhavyashah125.wordpress.com/

Contacting Me
E-mail Address: bhavya.shah125@gmail.com
Follow me on Twitter @BhavyaShah125 or www.twitter.com/BhavyaShah125
Mobile Number: +91 7506221750

**Comment by derekriemer** [comment]
Gotcha.  It is reliable. How common of a use case is this? I can try testing with your str soon, but I've seen this with somewhat large tables in the while. Have you tried with different document formatting settings disabled?

**Comment by Adriani90** [comment]
@sumandogra, @bhavyashah I have seens this issue multiple times. however, disabling pages in document formating settings and changing the view in MS word to "web layout" or so led to major improvements in performance. Did you try the suggestions above from Derek? Could you please test with last NVDA Alpha in Word 2016 and report if you still have this issue?

**Comment by Adriani90** [comment]
So it seems a crash or freeze is not reproducible here. I cannot reproduce any crash either. There are other issues dealing with performance lags in MS Word. I am closing this specific one as works for me. If you still have this crash problem, please comment on this issue anc we can reopen it. Thanks.
